content/en/docs/tutorials/dashboard-branding.md

```diff
@@ -7,23 +7,40 @@ keywords: [dashboard, branding, logo, colors]
 weight: 12
 ---
 
-Dashboard helps you to present to your stakeholders new device features as well as explain how this IoT framework works. To make it more convincing, basic templating functionality was introduced.
+In the plgd hub, customization of the UI for your company is possible. This includes the ability to modify the logo, colors, and even switch to your own preferred theme.
 
-## Colors change
+## Theme
 
-All colors are defined in one [`colors.scss`](https://github.com/plgd-dev/hub/tree/main/http-gateway/web/src/common/styles/colors.scss) file. Changing one of the colors will have an impact on all parts of the application.
+To create your own theme, follow these steps:
 
-## Logo change
+1. Clone the repository by running the following command:
 
-You can change the logo of the application by replacing these files:
+    ```sh
+    git clone --recursive --branch v2.15.0 https://github.com/plgd-dev/hub.git
```

Comment:
any reason to force version v2.15.0 and not main?

Reply:
Because it uses submodule and the revision could be changed by each version.